src/MatrixTypes_Trilinos.f90

```diff
@@ -397,7 +400,15 @@ SUBROUTINE matvec_TrilinosVector(thisMatrix,trans,alpha,x,beta,y,uplo,diag,incx_
             modName//'::'//myName//' - This interface is not available.')
       ENDSELECT
     ENDSUBROUTINE matvec_TrilinosVector
-    
+!
+!-------------------------------------------------------------------------------
+!> @brief zero all the elements of the matrix
+!> @param matrix declare the matrix type to act on
+!>
+!>
+    SUBROUTINE zeroentries_TrilinosMatrixType(matrix)
+      CLASS(TrilinosMatrixType),INTENT(INOUT) :: matrix
```

Comment:
Implement and test

Reply:
I don't believe for now there's a need for implementing this. Since the problem is only with PETSC.

Reply:
No need for this, trilinos reset the matrix when the matrix is assembled.